### PR TITLE
NCL-1882 UI for alignment reports: show products where an artifact wa…

### PR DESCRIPTION
**Pull request by matedo1**
…s shipped

### CODE REVIEW

File: ui/app/common/restclient/services/ReportDAO.js [hunk 1]
```diff
@@ -44,6 +44,20 @@
         });
       };
 
+      resource.getProductsByGAV = function (groupId, artifactId, version) {
+        return Configuration.then(function (config) {
+          return $http.get(config.dependencyAnalyzerReportsURL + '/v-0.4/listings/whitelist/artifacts/gav', {
```
Comment (vibe13) [inline ui/app/common/restclient/services/ReportDAO.js]: What about keeping all the endpoints in the config.json file, like getWhitelistProductArtifacts ?

**Comment by vibe13** [comment]
@matedo1 beside the small comment, looks good to me!

**Comment by matedo1** [comment]
Thanks @vibe13 , that's good point, I created new commit & pushed it, feel free to merge it if you are ok with that now

**Comment by vibe13** [comment]
@matedo1  merged, thanks!

**Pull request merged**